examples/integrations/nuance_pin/README.md

```diff
@@ -0,0 +1,186 @@
+```
+
+Next we must place the Nuance PIN `ai_service` wheel in the `nuance_pin/lib` folder. This would have been obtained
+in step 3 of of the [prerequisites](#prerequisites).
+
+### Running the Example App in the Container
+
+Now we are ready to build and start the container that runs our MONAI app as a Nuance service.
+```bash
+docker-compose up --build
+```
+
+If the build is successful the a service will start on `localhost:5000`. We can verify the service is running
+by issuing a "live" request such as
+```bash
+curl -v http://localhost:5000/aiservice/2/live && echo ""
+```
+The issued command should return the developer, app, and version of the deployed example app.
+
+Now we can run the example app with the example spleen data as the payload using Nuance PIN AI Service Test
+(`AiSvcTest`) utility obtained with the Nuance PIN SDK.
+```bash
+# create a virtual environment and activate it
+python3 -m venv /opt/venv
+. /opt/venv/bin/activate
+
+# install AiSvcTest
+pip install AiSvcTest-<version>-py3-none-any.whl
+
+# create an output directory for the inference results
+mkdir -p ~/Downloads/dcm/out
+
+# run AiSvcTest with spleen dicom payload
+python -m AiSvcTest -i ~/Downloads/dcm -o ~/Downloads/dcm/out -s http://localhost:5000 -V 2 -k
+```
+
+### Running the Example App on the Host
+
+Alternatively the user may choose to run the Nuance PIn service directly on the host. For this we must install the following:
+- Nuance PIN AI Serivce libraries
+- Libraries in the `requirements.txt`
+
+```bash
+# create a virtual environment and activate it
+python3 -m venv /opt/venv
+. /opt/venv/bin/activate
+
+# install Nuance Ai Service
+pip install ai_service-<version>-py3-none-any.whl
+
+# install requirements
+pip install -r requirements.txt
+
+# run the service
+export AI_PARTNER_NAME=NVIDIA
+export AI_SVC_NAME=ai_spleen_seg_app
+export AI_SVC_VERSION=0.1.0
+export AI_MODEL_PATH=model/model.ts
+export MONAI_APP_CLASSPATH=app.spleen_seg.AISpleenSegApp
+export PYTHONPATH=$PYTHONPATH:.
+python app_wrapper.py
+```
+
+Now we can issue a "live" request to check whether the service is running
+```bash
+curl -v http://localhost:5000/aiservice/2/live && echo ""
+```
+As we did in the last section, we can now run the example app with the example spleen data as the payload using Nuance PIN AI Service Test
+(`AiSvcTest`) utility obtained with the Nuance PIN SDK.
+```bash
+. /opt/venv/bin/activate
+
+# install AiSvcTest
+pip install AiSvcTest-<version>-py3-none-any.whl
+
+# create an output directory for the inference results
+mkdir -p ~/Downloads/dcm/out
+
+# run AiSvcTest with spleen dicom payload
+python -m AiSvcTest -i ~/Downloads/dcm -o ~/Downloads/dcm/out -s http://localhost:5000 -V 2 -k
+```
+
+### Bring Your Own MONAI App
+
+This example integration may be modified to fit any existing MONAI app, however, they may be caveats.
+
+Nuance PIN requires all artifacts present in the output folder to be also added into the `resultsManifest.json` output file
```

Comment:
Is it `resultsManifest.json` or `resultManifest.json` (s or no s), both are used here?